extension/lib/addProductsToCart.js

```diff
@@ -23,12 +23,13 @@ module.exports = async (context, input) => {
    * But that lead to the first add-to-cart call on an empty cart not working.
    */
 
-  let shopgateCartPipeline = ShopgateCartPipeline.create(context)
+  const shopgateCartPipeline = ShopgateCartPipeline.create(context)
   try {
-    return await shopgateCartPipeline.addProducts(input.products).catch((err) => {
+    await shopgateCartPipeline.addProducts(input.products).catch((err) => {
       if (err.code === 422) {
         const message = new ShopgateCartMessage('error', 'This product is not available anymore', 422)
         return {
+          cartId: '',
           messages: [message.toJson()]
```

Comment:
this is actually what I mentioned in the daily for @awesselburg to fix for ticket https://jira.shopgate.guru/browse/BIGC-811, these messages no longer exist in the pipeline definitions, so they need to be removed, along with this whole solution. I just added my stuff so that it's not breaking right now.